src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -459,11 +459,13 @@ object Parser {
         selectStmt |
         block |
         simpleStmt |
+        labeledStmt |
+        expressionStmt |
         emptyStmt
 
 
     lazy val simpleStmt: Parser[PSimpleStmt] =
-      sendStmt | assignmentWithOp | assignment | shortVarDecl | expressionStmt
+      sendStmt | assignmentWithOp | assignment | shortVarDecl // expressionStmt is parsed separately
```

Comment:
was this change necessary due to trying to parse `labelDef` as an expression?

Reply:
yes